Makefile

```diff
@@ -805,6 +829,40 @@ deb: install themispp_install themis_jni_install
 
 	@find $(BIN_PATH) -name \*.deb
 
+# TODO: Add Themis as dependency?
+# TODO: Put inside `deb` target?
+# Use builtin feature of fpm to create a .deb package from a Python package dir.
+# Dependencies are automatically added, i.e. PyThemis depends on `six`, so fpm will add `python3-six` to deps.
+pythemis_deb: DEB_ARCHITECTURE = all
+pythemis_deb: DESTDIR = $(BIN_PATH)/deb/pythemis_root
+pythemis_deb:
+	@mkdir -p $(BIN_PATH)/deb
+	@fpm --input-type python \
+		 --output-type deb \
+		 --python-bin=python3 \
+		 --python-package-name-prefix=python3 \
+		 --name python3-pythemis \
+		 --license $(LICENSE_NAME) \
+		 --url '$(COSSACKLABS_URL)' \
+		 --description '$(SHORT_DESCRIPTION)' \
+		 --maintainer $(MAINTAINER) \
+		 --package $(BIN_PATH)/deb/python3-pythemis_$(NAME_SUFFIX) \
+		 --architecture $(DEB_ARCHITECTURE) \
+		 --version $(VERSION)+$(OS_CODENAME) \
+		 --depends python3 \
```

Comment:
we need add here `libthemis` dependency too

Reply:
This also forces user to have Themis itself installed in system as `.deb` package. And it's a bit of an issue imo, because one needs to either add needed repos and install from them or use `make deb` which currently not only builds Themis itself, but also C++ and JNI wrappers. And while I see no problems with the first one, 'cause everybody got a C++ compiler, missing `jni.h` will cause errors and it will fail. So I think it's worth separating wrappers to separate targets like `deb_themispp` and `deb_jni`. To retain backward compatibility we could also move Themis packages to separate targets and make `deb` depend on all of them to work like before.

Reply:
> This also forces user to have Themis itself installed in system as .deb package.

yes, it's okay and expected behavior.

> And it's a bit of an issue imo, because one needs to either add needed repos and install from them or use make deb which currently not only builds Themis itself, but also C++ and JNI wrappers. 

it's okay that it need to install dependencies. If someone wants to install pythemis from the sources, it need to install libthemis too. And as you mentioned, user can do it from the package repository or build with makefile only if he need libthemis from the sources. Yes, for now it requires to compile themispp and themis jni due to `deb` is general target that builds all at once and we don't have `deb_libthemis` or `deb_core` that will not build anything else except core lib. When we will add this target, he will be able to pack deb packet for pythemis and be able to pack .deb libthemis from the sources. Adding one more target to build libthemis deb packet is out of scope this PR and I think we can do it in the separate. But we must specify dependency in the deb metadata for pythemis.